cmd/lncli/commands.go

```diff
@@ -1276,8 +1320,26 @@ func unlock(ctx *cli.Context) error {
 	}
 	fmt.Println()
 
+	args := ctx.Args()
+
+	// Parse the optional recovery window if it is specified. By default,
+	// the recovery window will be 0, indicating no lookahead should be
```

Comment:
some places it says it should be non-zero, here it says it can be zero. What's correct?

Reply:
Recovery window of 0 indicates no-lookahead, and no recovery will be attempted. 0 is a valid parameter, unless the user is actually trying to attempting to recover funds